tools/clang/test/DXC/extra_output.test

```diff
@@ -0,0 +1,8 @@
+// Test extra DXC outputs together
+// RUN: %dxc /T ps_6_0 %S/Inputs/smoke.hlsl /DDX12 /Dcheck_warning /Fh %t.h /Vn g_myvar /Fc %t.ll /Fo %t.cso /Fre %t.reflection /Frs %t.rootsig /Fe %t.err && FileCheck --input-file=%t.h %s --check-prefix=HEADER && FileCheck --input-file=%t.ll %s --check-prefix=IR
```

Comment:
This is the place that was of concern. The concern is not that this won't work, it clearly will, but that we don't understand why having it on two lines as you tried earlier didn't work. Not understanding that worries me. That said, perhaps it's not important at this point. @llvm-beanz, what do you think?

Reply:
Yea, this is exactly where I’m concerned. We should investigate why this doesn’t work on separate lines.

Reply:
That is issue of lit itself, not about this PR which just move some test to lit.
We could do that in anther PR.

Reply:
Issue created for check why | not work https://github.com/microsoft/DirectXShaderCompiler/issues/4956

Reply:
I think we can close this issue based on https://www.baeldung.com/linux/piped-commands-order where it says, "
The shell starts the second process while the first process is still running. Therefore, the processes run concurrently."
So FileCheck will start while dxc has not finish generating the temp files.
'|' only works for things like stdin.

Reply:
Yea, so the feedback here is, let's put these on separate run lines so that they are easier to read.